event-catalog/eventcatalog.config.js

```diff
@@ -0,0 +1,50 @@
+module.exports = {
+  title: 'EventCatalog - Insurance Claims Processing Application',
+  tagline: 'Discover, Explore and Document your Event Driven Architectures',
+  organizationName: 'Amazon Web Services',
+  projectName: 'Event Catalog',
+  editUrl: 'https://github.com/mythilias/serverless-eda-insurance-claims-processing/tree/event-catalog-new/event-catalog',
+  trailingSlash: true,
+  primaryCTA: {
+    label: 'Explore Events',
+    href: '/events'
+  },
+  //: {
+  //  label: 'Getting Started',
+  //  href:"https://www.eventcatalog.dev/"
+  //},
+  logo: {
+    alt: 'EventCatalog Logo',
+    // found in the public dir
+    src: 'logo.svg',
+  },
+  headerLinks: [
+    { label: 'Events', href: '/events'},
+    { label: 'Services', href: '/services' },
+    { label: 'Domains', href: '/domains'},
+    { label: 'Users', href: '/users'},
+    { label: 'Visualiser', href: '/visualiser' },
+    { label: '3D Node Graph', href: '/overview' },
+  ],
+  footerLinks: [
+    { label: 'Events', href: '/events' },
+    { label: 'Services', href: '/services' },
+    { label: 'Visualiser', href: '/visualiser' },
+    { label: '3D Node Graph', href: '/overview' },
+    { label: 'GitHub', href: 'https://github.com/mythilias/serverless-eda-insurance-claims-processing/tree/event-catalog-new/event-catalog' }
+  ],
+  users: [
+    {
+      id: 'dhiraj',
+      name: 'Dhiraj Mahapatro',
+      avatarUrl: 'https://randomuser.me/api/portraits/lego/5.jpg',
+      role: 'Principal SSA',
```

Comment:
SSA will be an acronym, no one would understand. Can you please change it to "Principal Specialist Solutions Architect, Serverless" ?

Reply:
updated